subsys/modem/modem_cmux.c

```diff
@@ -816,6 +816,14 @@ static void modem_cmux_process_received_byte(struct modem_cmux *cmux, uint8_t by
 		/* Get last 8 bits of data length */
 		cmux->frame.data_len |= ((uint16_t)byte) << 7;
 
+		if (cmux->frame.data_len > cmux->receive_buf_size) {
+			LOG_ERR("Indicated frame data length %u exceeds receive buffer size %u",
```

Comment:
Why `Indicated`? Wouldn't it rather be `Received`?

Reply:
I guess one could also say `Invalid` as well?

Reply:
Ah, now I think I see what you mean with `Indicated`. Well, whatever, this was a minor remark anyway.